diktat-rules/src/main/kotlin/generated/WarningNames.kt

```diff
@@ -77,6 +77,8 @@ object WarningNames {
 
     const val KDOC_NO_DEPRECATED_TAG: String = "KDOC_NO_DEPRECATED_TAG"
 
+    const val WRONG_KDOC_PARAM: String = "WRONG_KDOC_PARAM"
```

Comment:
I think it's not worth creating a separate warning, please add this case to `KDOC_WITHOUT_PARAM_TAG` just with different `freeText` in error message.